content/applications/sales/crm/performance/expected_revenue_report.rst

```diff
@@ -0,0 +1,200 @@
+
+.. image:: expected_revenue_report/match-all-filters.png
+   :align: center
+   :alt: Emphasis on the match all filters option on the Add Custom Filter pop-up window.
+
+At the bottom of the :guilabel:`Add Custom Filter` form, click :guilabel:`Add`.
+
+View options
+------------
+
+The expected revenue report benefits from utilizing multiple views. The default graph view can be
+used to identify which salespeople are expected to bring in the most revenue, while the list view
+and pivot view provide more detail on specific deals.
+
+.. tabs::
+
+   .. tab:: Graph view
+
+      The *graph view* is used to visualize data, and is beneficial in identifying patterns and
+      trends.
+
+      *Bar charts* are used to show the distribution of data across several categories or among
+      several salespeople.
+
+      *Line charts* are useful to show changing trends over a period of time.
+
+      *Pie charts* are useful to show the distribution, or comparison, of data among a small number
+      of categories or salespeople, specifically how they form the meaningful part of a whole
+      picture.
+
+      The default view for the expected revenue report is the bar chart, stacked. To change to a
+      different graph view, click one of the icons at the top-left of the report. While both the
+      line chart and bar chart are available in stacked view, the pie chart is not.
+
+      .. figure:: expected_revenue_report/graph-view-icons.png
+         :align: center
+         :alt: Close up view of the graph icons on the Pipeline analysis report in the CRM app.
+
+         Graph view icons in order: bar chart, line chart, pie chart, stacked.
+
+   .. tab:: List view
+
+      The *list view* provides a list of all leads that are expected to close by the designated
+      date. Clicking on a lead in list view opens the record for detailed analysis, but many
+      insights can be gleaned from the basic view.
+
+      To switch to the list view, click the :guilabel:`≣ (list)` icon at the top-right of the
+      report.
+
+      .. image:: expected_revenue_report/list-icon.png
+         :align: center
+         :alt: Close up of the list view icon in the CRM app.
+
+      To add additional metrics to the report, click the *additional options menu* indicated by the
+      :guilabel:`toggle` icon at the top-right of the list.
+
+      .. figure:: expected_revenue_report/toggle-icon.png
+         :align: center
+         :alt: Close up of the toggle icon in the CRM app.
+
+         Clicking the toggle icon in *list view* opens the *additional options menu*.
+
+      Select any additional metrics from the drop-down menu to add them to the list view. Some
+      options that may be useful are :guilabel:`Expected Closing` and :guilabel:`Probability`.
+
+   .. tab:: Pivot view
+
+      The *pivot view* arranges all leads that are expected to close by the designated date into a
+      table, with salespeople along the Y axis and stages along the X axis.
```

Comment:
> with salespeople along the Y axis

Doesn't appear to necessarily be true (see screenshot below):
 ![Odoo-Pipeline-Analysis(1)](https://github.com/odoo/documentation/assets/5496809/d78e009e-df13-43a2-862b-9eb766c08dc9)

We'll want to specify what the *default* is for the pivot table and how these axis values can be manipulated via the parameters in the popover menus.

As well, I'd argue _pivot table_ is going to be the **most important** view since it allows sales managers to format data quickly and nicely before injecting into a spreadsheet of some kind (hopefully Odoo _Spreadsheet_ 😉) for further analysis.